contracts/DelFiPrice.sol

```diff
@@ -123,4 +123,4 @@ contract DelFiPrice is OpenOracleView {
         }
         return array;
     }
-}
+}
```

Comment:
This no new line thing is annoying, any ideas how to disable that?

Reply:
I think you'd have to add a newline